include/deal.II/matrix_free/cuda_matrix_free.templates.h

```diff
@@ -333,33 +191,50 @@ namespace CUDAWrappers
       // TODO this should be a templated parameter.
       const unsigned int n_dofs_1d = fe_degree + 1;
 
-      if (data->parallelization_scheme ==
-          MatrixFree<dim, Number>::parallel_in_elem)
-        {
```

Comment:
I removed the choice because it was not tested and I want to expose the parallelism differently when using Kokkos. I want to expose the nested parallel loops and let Kokkos decide how to parallelize the loops.